src/VerifyHandler.php

```diff
@@ -185,7 +185,8 @@ protected function getCredentialRequestOptions(
             return $source->getPublicKeyCredentialDescriptor();
         }, $validCredentials ?? []);
 
-        $options = new PublicKeyCredentialRequestOptions(random_bytes(32), 40000);
+        $options = new PublicKeyCredentialRequestOptions((string) random_bytes(32));
+        $options->setTimeout(4000);
```

Comment:
```suggestion
        $options = new PublicKeyCredentialRequestOptions((string) random_bytes(32), timeout: 4000);
```
We can use named arguments here, which is preferred since the `setTimeout` method is marked as deprecated.

Reply:
Updated

Reply:
v4.4 doesn't have `$timeout` parameter in constructor.